poc/flp_generic.py

```diff
@@ -640,6 +641,140 @@ def test_vec_set_type_param(self, test_vec):
         return ['length', 'chunk_length']
 
 
+class MultiHotHistogram(Valid):
+    """
+    A validity circuit that checks each Client's measurement is a bit vector,
+    and has at most `max_num_bits` 1s.
+
+    In order to check the Client measurement `x` has at most `max_num_bits` 1s,
```

Comment:
Self-consistency: We call this the "count check" in the circuit.
```suggestion
    In order to check the Client measurement `x` has at most `max_count` 1s,
```